docs/index.html

```diff
@@ -45,17 +45,26 @@ <h1 class="display-5">Ballerina Cloud Performance Stats</h1>
     <div class="row ">
         <div class="col-12 col-md-9">
             <h5>Purpose of the framework</h5>
-            <p>This performance framework was developed to get an idea of how ballerina performance changes with nightly builds and how ballerina performs compared to other languages in different use cases. </p>
+            <p>This performance framework was developed to get an idea of how Ballerina performance changes with nightly builds and how Ballerina performs compared to other languages in different use cases. </p>
 
-            <h5>Test Execution</h5>
+            <h5>Test Execution for Hello World Sample</h5>
             <ol>
-                <li>Create a fresh single node cluster(Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx ingress.</li>
-                <li>Deploy the application pod into the cluster (Hello world service written by different languages with 1 cpu, 512mb memory resource limits).</li>
-                <li>Deploy the JMeter pod and execute the benchmark and then commit the results to Github. (Concurrent Users - 50)</li>
+                <li>Create a fresh single node (Standard_F4s_v2) Kubernetes cluster in Azure and installs Nginx Ingress.</li>
+                <li>For each language hello world sample</li>
```

Comment:
This seems to be incomplete.

Reply:
This specifies a loop.  Modified to make it bit more readable. Please see if its okay